yarn.lock

```diff
@@ -5,6 +5,7 @@
 "@babel/cli@7.0.0":
```

Comment:
Please don't commit this file as the changes come from the fact you're using a more recent `yarn` than what we used to use. Because in this PR there's nothing about this file, let's not change it now. I'll file a separate PR to do it.

You can remove this commit using an interactive rebase, with `git rebase -i`. You can learn more about this command in [the git book](https://git-scm.com/book/en/v2/Git-Tools-Rewriting-History). Removing a commit is as easy as deleting the line for this commit in the list.